From 2ce255f593426cd5943da0a7db73cda438318d1d Mon Sep 17 00:00:00 2001
From: Cristina Ciocan <cristina-mihaela.ciocan@nxp.com>
Date: Thu, 22 Jun 2017 17:46:59 +0300
Subject: [PATCH 2002/5242] MLK-15125: arm: dts: imx6sx: Fix duplicate name in
 lcdif

commit  ed7a3fd022cfc95933ac8b6196f3744a698c2999 from
https://source.codeaurora.org/external/imx/linux-imx.git

lcdif2 node has a property called "display" and a subnode that is also
called "display", leading to an OF duplicate warning at boot time.

Fix this by changing the subnode's name to "display@1".

Signed-off-by: Cristina Ciocan <cristina-mihaela.ciocan@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6sx-sabreauto.dts |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/imx6sx-sabreauto.dts b/arch/arm/boot/dts/imx6sx-sabreauto.dts
index 9e58f05..5f7cde2 100644
--- a/arch/arm/boot/dts/imx6sx-sabreauto.dts
+++ b/arch/arm/boot/dts/imx6sx-sabreauto.dts
@@ -526,7 +526,7 @@
 	disp-dev = "ldb";
 	status = "okay";
 
-	display1: display {
+	display1: display@1 {
 		bits-per-pixel = <16>;
 		bus-width = <18>;
 	};
-- 
1.7.9.5

